openfisca_france_local/departements/eure_et_loir/aide-menagere.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+from openfisca_france.model.base import Variable, Individu, MONTH
+from openfisca_france.model.caracteristiques_socio_demographiques.demographie import ressortissant_eee
+from openfisca_france.model.prestations.autonomie import TypesGir
+
+
+class eure_et_loir_aide_menagere_personne_agee(Variable):
```

Comment:
L'objet de cette variable étant de calculer l'éligibilité, la convention est de faire figurer cet aspect dans le nom.
Dans ce dépôt, c'est indiqué après la zone géographique. Renommer la variable pour le faire figurer ?

```suggestion
class eure_et_loir_eligibilite_aide_menagere_personne_agee(Variable):
```

> Remarques auxiliaires : 
> L'éligibilité est plutôt en suffixe sur `openfisca-france` ; nous avons donc une petite incohérence là-dessus 😕 
> D'après le RDAS, c'est d'abord une aide sociale à domicile puis, elle est mise en oeuvre au travers de l'aide-ménagère à domicile et, finalement, est spécifique aux personnes âgées. Ca me semble un peu long à faire figurer dans le nommage 😅